modules/installation-azure-subscription-tenant-id.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+:_content-type: PROCEDURE
+[id="installation-azure-subscription-tenant-id_{context}"]
+= Identifying the subscription and tenant ID
+
+The installation program requires the subscription and tenant IDs that are associated with your Azure account. You can use the Azure CLI to gather this information.
+
+.Prerequisites
+
+* You have installed or updated the link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-yum?view=azure-cli-latest[Azure CLI].
+
+.Procedure
+
+. Log in to the Azure CLI:
++
+[source,terminal]
+----
+$ az login
+----
+
+. Ensure that you are using the right subscription:
+
+.. View the list of available accounts and record the `tenantId` value for the
+subscription you want to use for your cluster:
++
+[source,terminal]
+----
+$ az account list --refresh
+----
++
+.Example output
+[source,terminal]
+----
+[
+  {
+    "cloudName": "AzureCloud",
+    "id": "9bab1460-96d5-40b3-a78e-17b15e978a80",
+    "isDefault": true,
+    "name": "Subscription Name",
+    "state": "Enabled",
+    "tenantId": "6057c7e9-b3ae-489d-a54e-de3f6bf6a8ee",
+    "user": {
+      "name": "you@example.com",
+      "type": "user"
+    }
+  }
+]
+----
+
+.. View your active account details and confirm that the `tenantId` value matches
+the subscription you want to use:
++
+[source,terminal]
+----
+$ az account show
+----
++
+.Example output
+[source,terminal]
+----
+{
+  "environmentName": "AzureCloud",
+  "id": "9bab1460-96d5-40b3-a78e-17b15e978a80",
+  "isDefault": true,
+  "name": "Subscription Name",
+  "state": "Enabled",
+  "tenantId": "6057c7e9-b3ae-489d-a54e-de3f6bf6a8ee", <1>
+  "user": {
+    "name": "you@example.com",
+    "type": "user"
+  }
+}
+----
+<1> Ensure that the value of the `tenantId` parameter is the correct subscription ID.
+
+.. If you are not using the right subscription, change the active subscription:
++
+[source,terminal]
+----
+$ az account set -s <subscription_id> <1>
+----
+<1> Specify the subscription ID.
+
+.. Verify the subscription ID update:
++
+[source,terminal]
+----
+$ az account show
+----
++
+.Example output
+[source,terminal]
+----
+{
+  "environmentName": "AzureCloud",
+  "id": "33212d16-bdf6-45cb-b038-f6565b61edda",
+  "isDefault": true,
+  "name": "Subscription Name",
+  "state": "Enabled",
+  "tenantId": "8049c7e9-c3de-762d-a54e-dc3f6be6a7ee",
+  "user": {
+    "name": "you@example.com",
+    "type": "user"
+  }
+}
+----
+
+. Record the `tenantId` and `id` parameter values from the output. You need these values during the {product-title} installation.
```

Comment:
As part of our demo this week @r4f4 , let's use part of the time to revisit this procedure. I am presently reusing existing content from the published docs [1], but even without access to an account an the Azure CLI, something does not feel correct about it. For example, steps 2a and 2b in the published output seem identical. 
If possible, it would be good to tighten this writing up while we are here.

[1] https://docs.openshift.com/container-platform/4.13/installing/installing_azure/installing-azure-account.html#installation-azure-service-principal_installing-azure-account

Reply:
I guess they are the same in this case because there is only one account/subscription involved. Here is an example output where 2a would return more than one result:
```
$ az account list
[
  {
    "cloudName": "WWT",
    "id": "<Redacted Id1>",
    "isDefault": true,
    "name": "Redhat-2",
    "state": "Enabled",
    "tenantId": "<Redacted T1>",
    "user": {
      "name": "redhatuser28284@wwtatc.com",
      "type": "user"
    }
  },
  {
    "cloudName": "WWT",
    "id": "<Redacted Id2>",
    "isDefault": false,
    "name": "Redhat",
    "state": "Enabled",
    "tenantId": "<Redacted T1>",
    "user": {
      "name": "XXX@wwtatc.com",
      "type": "user"
    }
  }
]
```
But 2b shows:
```
$ az account show
{
  "environmentName": "WWT",
  "id": "<Redacted Id1>",
  "isDefault": true,
  "name": "Redhat-2",
  "state": "Enabled",
  "tenantId": "<Redacted T1>",
  "user": {
    "name": "XXX@wwtatc.com",
    "type": "user"
  }
}
```